src/Discretization/ExponentiationModule.jl

```diff
@@ -1,8 +1,30 @@
-using LinearAlgebra: checksquare
+"""
+Interface to matrix exponential backends of different kinds.
 
-# ==========================
-# Exponentiation functions
-# ==========================
+Includes common integral computations arising in the discretization of linear differential equations
+using matrix methods. For applications see e.g. [1] and references therein.
+
+[1] Conservative Time Discretization: A Comparative Study.
+    Marcelo Forets and Christian Schilling (2022).
+    Proceedings of the 17th International Conference on integrated Formal Methods (iFM),
+    LNCS, vol. 13274, pp. 149-167. doi: 10.1007/978-3-031-07727-2_9, arXiv: 2111.01454.
+"""
+module ExponentiationModule
+
+using StaticArrays
+using MathematicalSystems
+using ReachabilityBase.Require
+using LinearAlgebra: checksquare, I, Diagonal
+using SparseArrays
+using IntervalArithmetic
+using IntervalMatrices
+using LazySets
+
+export BaseExp, BaseExpAlg, IntervalExpAlg, LazyExpAlg, PadeExpAlg, _exp, _alias, _elementwise_abs,
+       _Φ₂, _Φ₁, _Φ₁_u, load_expokit_pade, _interval_matrix, _Cδ
```

Comment:
We should not export functions starting with `_`. If we need them, we should remove the `_` or import them explicitly.